conjur/api/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+api
+
+Package containing classes that responsible to communicate with conjur server.
```

Comment:
that are responsible for communicating with the Conjur server

Reply:
Done